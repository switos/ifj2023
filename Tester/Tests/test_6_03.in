func getNumber() -> Int {
    // Some code here
    return
}
// Empty return in a function that should return an int
