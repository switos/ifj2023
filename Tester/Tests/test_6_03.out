Expected Output:

Expected Return Code: 6
